lib/utils.server.ts

```
@@ -5,15 +5,17 @@ import {
 } from '@/lib/constants';
 
-import type { DBMessage } from './db/schema';
 import { createOpenRouter } from '@openrouter/ai-sdk-provider';
 import {
+  CoreUserMessage,
   generateText,
+  UIMessage,
   type TextPart,
   type ToolCallPart,
   type ToolResultPart,
 } from 'ai';
```

Biome (1.9.4)

[error] Some named imports are only used as types.

This import is only used as a type.

Importing the types with import type ensures that they are removed by the compilers and avoids loading unnecessary modules.
Safe fix: Add inline type keywords.


(lint/style/useImportType)